src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -1,13 +1,24 @@
+import { BusinessImageProviderAggregator } from '../../providers/data_providers/businessImageProviderAggregator';
+import {
+	ExtractionConfig,
+	getExtractionProvider,
+} from '../../providers/extraction_providers/extractionProviderFactory';
+import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
 import { BusinessInferences, BusinessInfoInput } from '../generatedTypes';
 
 export async function getBusinessInfo(businessInfo: BusinessInfoInput): Promise<BusinessInferences> {
-	return {
-		servesAlcohol: {
-			insight: false,
-			confidence: 0.5,
-			evidence: {
-				images: [],
-			},
-		},
-	};
+	if (!process.env.EXTRACTION_CONFIG) throw new Error('Undefined environment variable EXTRACTION_CONFIG');
+	if (!process.env.INTERPRETATION_CONFIG)
+		throw new Error('Undefined environment variable INTERPRETATION_CONFIG');
+
+	const extractionConfig: ExtractionConfig = JSON.parse(process.env.EXTRACTION_CONFIG);
+	const interpretationConfig: InterpretationConfig = JSON.parse(process.env.INTERPRETATION_CONFIG);
+
+	const images = await new BusinessImageProviderAggregator().getImages(businessInfo);
+	const extractions = getExtractionProvider(extractionConfig).extract(images);
+	const interpretations = await getInterpretationProvider(interpretationConfig).then((provider) =>
```

Comment:
`getInterpretationProvider` is a mildly expensive call to make, so "caching" would be nice

The usual pattern to do this is dependency injection; we pass the already-configured providers as an argument into this function. Subsequent invokings of this function would also use the same providers.